server/server.js

```diff
@@ -72,7 +75,22 @@ app.post("/api/addWorkout", async (req, res) => {
       "INSERT INTO workouts(user_id, video_id, target_area, exercises) VALUES($1, $2, $3, $4) RETURNING*",
       [userId, videoId, targetArea, exercises]
     );
-    res.json(workout);
+    res.json(workout[0]);
+  } catch (error) {
+    console.log(error.message);
+  }
+});
+
+// update a specific workout
+app.put("/api/updateWorkout", async (req, res) => {
+  try {
+    const { videoId, userId, targetArea, exercises } = req.body;
```

Comment:
Hmm curious if your workouts have a unique id? like just a workout Id?

And if so - why the decision to identify them via (userId, videoId) combo rather than the workoutId?

(not saying one way is right or wrong, but would love to hear the thought process behind this decision)